GATracker.swift

```diff
@@ -111,7 +108,7 @@ class GATracker {
                 else {
                         if (error != nil) {
```

Comment:
Optionals should not be handled like this. Use `if let error = error` or some other optional binding mechanism.